Keas.Mvc/ClientApp/components/Keys/EditKey.tsx

```diff
@@ -14,7 +15,8 @@ interface IProps {
 }
 
 interface IState {
-  error: string;
+  errorMessage: string;
+  errorPath: string;
```

Comment:
@laholstege what do you think about combining this into like a `IValidationError` object or something, that way you can keep the state clean and only pass one object, and plus it also leaves room for other error types.  You might even just be able to use the yup native `ValidationError` object too, or something similar to it.